### PR TITLE
Spline Sans Mono: Version 1.001 added

### PR DESCRIPTION
**Pull request by RosaWagner**
0bc7e11: [gftools-packager] Spline Sans Mono: Version 1.000 added

* Spline Sans Mono Version 1.000 taken from the upstream repo https://github.com/SorkinType/SplineSansMono at commit https://github.com/SorkinType/SplineSansMono/commit/2f0ffb83d23a462e35a4d8781e6644f2c6a07b90.

### CODE REVIEW

File: ofl/splinesansmono/OFL.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-Copyright 2021 The Spline Sans Mono Project Authors (https://github.com/SorkinType/SplineSansMono)
+Copyright 20** The My Font Project Authors (https://github.com/googlefonts/My-Font-Repository)
```
Comment (RosaWagner) [inline ofl/splinesansmono/OFL.txt:1]: @bghryct It appears, the OFL.txt in the upstream repo is wrong

File: ofl/splinesansmono/METADATA.pb [hunk 2]
```diff
@@ -10,7 +10,7 @@ fonts {
   filename: "SplineSansMono[wght].ttf"
   post_script_name: "SplineSansMono-Regular"
   full_name: "Spline Sans Mono Regular"
-  copyright: "Copyright 2022 The Spline Sans Mono Project Authors (https://github.com/SorkinType/SplineSansMono)"
+  copyright: "Copyright 2022 The Spline Sans Project Authors (https://github.com/bghryct/SplineMono)"
```
Comment (RosaWagner) [inline ofl/splinesansmono/METADATA.pb:13]: @bghryct Check consistency of copyright mentions in font sources + repo

**Comment by bghryct** [comment]
Oh drat, the ofl should have the sorkintype url not the bghryct url. I can resolve this tomorrow I think.

**Comment by RosaWagner** [comment]
Updated

## Spline Sans Mono: Version 1.000 added

---
 46be919: [gftools-packager] Spline Sans Mono: Version 1.000 added

* Spline Sans Mono Version 1.000 taken from the upstream repo https://github.com/SorkinType/SplineSansMono at commit https://github.com/SorkinType/SplineSansMono/commit/81f3def3878b3608c589339abdce77711771bc5e.

File: ofl/splinesansmono/METADATA.pb (hunk 2 above)
Comment (RosaWagner) [reply ofl/splinesansmono/METADATA.pb:14]: @bghryct The copyright string in the font is still not the same as in the OFL license.

**Comment by RosaWagner** [comment]
@bghryct check the axis mapping, medium style is not linking design coordinates to proper user coordinates:

🔥 FAIL: Variable font weight coordinates must be multiples of 100. ([com.google.fonts/check/varfont_weight_instances](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/varfont_weight_instances))
🔥 FAIL Found a variable font instance with 'wght'=478.0. This should instead be a multiple of 100. [code: bad-coordinate]
🔥 FAIL: Check variable font instances have correct coordinate values ([com.google.fonts/check/varfont_instance_coordinates](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/varfont_instance_coordinates))
🔥 FAIL Instance "Medium" wght value is "478.0". It should be "500.0" [code: bad-coordinate]
🔥 FAIL Further info can be found in our spec https://github.com/googlefonts/gf-docs/tree/main/Spec#axes[](https://github.com/RosaWagner)[](https://github.com/apps/github-actions)[](https://github.com/apps/github-actions)[](https://github.com/apps/google-cla)[](https://github.com/RosaWagner)[](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)[](https://github.com/RosaWagner)[](https://github.com/gf-bot)[](https://github.com/bghryct)[](https://github.com/)

**Comment by RosaWagner** [comment]
Updated

## Spline Sans Mono: Version 1.000 added

---
 314afa4: [gftools-packager] Spline Sans Mono: Version 1.000 added

* Spline Sans Mono Version 1.000 taken from the upstream repo https://github.com/SorkinType/SplineSansMono at commit https://github.com/SorkinType/SplineSansMono/commit/fd94fa5f329b3b586fcda16a0547bca62b2269b3.

**Comment by RosaWagner** [comment]
Updated

## Spline Sans Mono: Version 1.002 added

---
 9bede6b: [gftools-packager] Spline Sans Mono: Version 1.002 added

* Spline Sans Mono Version 1.002 taken from the upstream repo https://github.com/SorkinType/SplineSansMono at commit https://github.com/SorkinType/SplineSansMono/commit/635218dc887e4f8f39f4c0ee8b95462df1be4866.

**Pull request merged**